src/test/java/com/rick/racing/domain/CarTest.java

```diff
@@ -0,0 +1,60 @@
+package com.rick.racing.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.rick.racing.controller.SettableCarMovingStrategy;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class CarTest {
+
+    private static final String CAR_NAME = "TSTR";
+
+    @Test
+    @DisplayName("인스턴스 정상 생성 확인")
+    public void create() {
+        final Car carA = Car.create(CAR_NAME);
+        final Car carB = Car.create(CAR_NAME);
+
+        assertThat(carA.equals(carB))
+            .isTrue();
```

Comment:
다음과 같이 한 줄로 표현해볼 수도 있을 것 같아요.
```suggestion
        assertThat(carA).isEqualTo(carB);
```